pkg/base1/cockpit.css

```diff
@@ -78,13 +78,13 @@ a {
 /* Break up sidebar in columns in smaller sizes*/
 
 @media (min-width: 992px) {
-    .cockpit-info-table-container .cockpit-info-table {
+    .info-table-ct-container .info-table-ct {
```

Comment:
I was trying to copy patternfly. They use ```xxx-pf``` after the initial class, and then subclasses go ```xxx-pf-yyy```. Take a look in patternfly-additions.css and search for ```-pf-``` you'll see lots of new stuff being done exactly this way.